stereo_node:
  ros__parameters:
    enable_depth: false

    image_width: 640
    image_height: 360
    frame_rate: 5

    left_format: MJPG
    left_port: /dev/video0
    left_device: Raspi

    right_format: MJPG
    right_port: /dev/video4
    right_device: Raspi
    
    imu_rate: 100
    
    frame_id: "camera_link"
    left_camera_frame: "left_camera_frame"
    right_camera_frame: "right_camera_frame"
    imu_frame: "imu_link"
    exposure_mode: "normal"
    gain: 1.0
    exposure_time: 0  # 0 for auto
    left_camera_info_url: "package://stereo_cam/config/left_camera_info.yaml"
    right_camera_info_url: "package://stereo_cam/config/right_camera_info.yaml"
    calibration_file: "package://stereo_cam/config/StereoCalibration.yaml"

  
    stereo:
      baseline: 0.06  # 60mm between cameras
      focal_length: 2822.10  # pixels
      principal_point: [1640.0, 1232.0]  # pixels 